src/main/java/com/cisco/trex/stateful/TRexAstfClient.java

```diff
@@ -474,7 +475,7 @@ public String getVersion() {
           try {
             AstfStatistics astfStatistics =
                 new ObjectMapper()
-                    .readValue(result.get(tgId.toString()).toString(), AstfStatistics.class);
+                    .readValue(Optional.ofNullable(result.get(tgId.toString())).map(t->t.toString()).orElse("{}"), AstfStatistics.class);
```

Comment:
it's not a good idea to create a "empty" result for non-appearing tgId, if it's no result of a specific, then considering leave it from the returned map